Cargo.toml

```diff
@@ -27,23 +27,23 @@ args = ["structopt"]
 color-eyre = "0.5"
 educe = "0.4"
 itertools = "0.9"
-openssh = "0.6"
+openssh = "0.7"
 rand = "0.7"
```

Comment:
Let's bump `rand` too while we're at it. Or in a separate PR if you prefer.

Reply:
Done.